x-pack/plugins/security_solution/public/timelines/components/timeline/body/events/event_column_view.test.tsx

```diff
@@ -145,14 +153,16 @@ describe('EventColumnView', () => {
   });
 
   test('it renders correct tooltip for NotesButton - timeline template', () => {
-    (useShallowEqualSelector as jest.Mock).mockReturnValue(TimelineType.template);
+    (useShallowEqualSelector as jest.Mock).mockReturnValue({
+      ...testTimeline,
+      timelineType: TimelineType.template,
+    });
 
     const wrapper = mount(<EventColumnView {...props} />, { wrappingComponent: TestProviders });
```

Comment:
Additionally, I know it is not in the scope exactly, but you can see if this file is quick to change from `enzyme` to `RTL`. Because these incremental changes are the only to get these done.